lib/uinta/formatter/util.ex

```diff
@@ -69,6 +69,12 @@ defmodule Uinta.Formatter.Util do
   @spec serialize(term()) :: String.t() | nil
   defp serialize(value) do
     cond do
+      is_list(value) && Enum.all?(value, &String.Chars.impl_for(&1)) ->
+        Enum.map(value, &to_string(&1))
+
+      is_list(value) && Enum.all?(value, &Inspect.impl_for(&1)) ->
+        Enum.map(value, &inspect(&1))
+
       String.Chars.impl_for(value) ->
```

Comment:
Do we want this up higher? I feel like if it's chardata we don't want to inspect it or return a list of strings. I don't know if there's a great check for chardata, though, because you have to traverse an entire nested list to determine if it's valid. So maybe a try-rescue?

Reply:
Good call. I've added some tests to help prevent breaking the behavior of handling various data types.

Reply:
It would be nice to preserve boolean, integer, and float values but I realize that could introduce breaking changes. This is a better short-term fix.

Reply:
Yeah we could solve those in the future with a guard clause or something, but this is probably fine for now